Functions.Templates/ProjectTemplate_v4.x/FSharp-Isolated/.template.config/template.json

```diff
@@ -61,6 +61,74 @@
             "type": "parameter",
             "defaultValue": "V4",
             "replaces": "AzureFunctionsVersionValue"
+        },
+        "FunctionsWorkerPackageVersion": {
+            "type": "generated",
+            "generator": "switch",
+            "replaces": "FunctionsWorkerPackageVersionValue",
+            "parameters": {
+                "datatype": "string",
+                "cases": [
+                    {
+                        "condition": "FrameworkShouldUseV1Dependencies",
+                        "value": "1.23.0"
+                    },
+                    {
+                        "value": "2.0.0"
+                    }
+                ]
+            }
+        },
+        "FunctionsAspNetCorePackageVersion": {
+            "type": "generated",
+            "generator": "switch",
+            "replaces": "FunctionsAspNetCorePackageVersionValue",
+            "parameters": {
+                "datatype": "string",
+                "cases": [
+                    {
+                        "condition": "FrameworkShouldUseV1Dependencies",
+                        "value": "1.3.2"
```

Comment:
The latest version is 1.3.3 - https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore/1.3.3. Similar update in `csharp-isolated/template.config` - https://github.com/Azure/azure-functions-templates/blob/96f239de0f22eaa3e5a9249319918f49199d7938/Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/.template.config/template.json#L121